static/js/modules/cycle-select.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+/* global window */
+
+var $ = require('jquery');
+var _ = require('underscore');
+var URI = require('URIjs');
+
+function CycleSelect(elm) {
+  this.$elm = $(elm);
+  this.$elm.on('change', this.handleChange.bind(this));
+}
+
+CycleSelect.build = function($elm) {
+  switch ($elm.data('cycle-location')) {
+  case 'query':
```

Comment:
Are the case statements supposed to be indented? I can't find the rule in the airbnb js styleguide but I'd expect it to be.